tests/test_grains.py

```diff
@@ -21,3 +25,10 @@ def test_gaussian_filter(small_array_grains: Grains, grain_config: dict) -> None
 
     assert isinstance(small_array_grains.images["gaussian_filtered"], np.ndarray)
     np.testing.assert_array_equal(small_array_grains.images["gaussian_filtered"], target)
+
+
+def test_random_grains(random_grains: Grains, caplog) -> None:
+    """Test errors raised when processing images without grains."""
+    # FIXME : Uncomment once filtering excludes all noise (grains < min_size) as we should find no grains.
+    # assert "No grains found" in caplog.text
+    assert True
```

Comment:
Is this to be fixed prior to merge, or post?

Reply:
Post, its contingent on @SylviaWhittle work which will include removing all "noise" when things are below a certain threshold.  Currently we _always_ get some grains found even on water samples. For example the `random` image used in places of this test suite results in the following HUGE number of grains being found which clearly doesn't make any sense!

![image](https://user-images.githubusercontent.com/17566406/176397691-bdfc2922-047b-4abf-abc9-77350bd2ce11.png)